setup/lib/utils.js

```diff
@@ -0,0 +1,74 @@
+const { setFailed } = require('@actions/core');
+const os = require('os');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const semver = require('semver');
+const core = require('@actions/core');
+
+// arch in [arm, x32, x64...] (https://nodejs.org/api/os.html#os_os_arch)
+// return value in [amd64, 386, arm]
+function mapArch(arch) {
+    const mappings = {
+        x32: '386',
+        x64: 'amd64'
+    };
+    return mappings[arch] || arch;
+}
+
+// os in [darwin, linux, win32...] (https://nodejs.org/api/os.html#os_os_platform)
+// return value in [darwin, linux, windows]
+function mapOS(os) {
```

Comment:
Can we change the name of the variable `os` because it is a built-in library name.

Reply:
@Two-Hearts, would this be a good item to add to the refactor list?

Reply:
> @Two-Hearts, would this be a good item to add to the refactor list?

@duffney Sure thing, because other PRs may need the same change.

Reply:
@Two-Hearts should I open this as an issue on the repo or do you have a running list for the refactor work?

Reply:
> @Two-Hearts should I open this as an issue on the repo or do you have a running list for the refactor work?

@duffney yeah, please open an issue on it. Thanks.

Reply:
Issue opened with a link to this thread. 

https://github.com/notaryproject/notation-action/issues/6